python/ingestion/census_to_bq.py

```diff
@@ -102,3 +105,43 @@ def write_population_by_race_to_bq(dataset, table_name, gcs_bucket, filename):
   except json.JSONDecodeError as err:
     msg = 'Unable to write to BigQuery due to improperly formatted data: {}'
     logging.error(msg.format(err))
+
+
+def write_household_income_to_bq(dataset, table_name, gcs_bucket, file_prefix):
+  """Fetches all SAIPE blobs from a GCS bucket and uploads to a single BQ table. Also does some preprocessing.
+
+     dataset: The BigQuery dataset to write to
+     table_name: The name of the BigQuery table to write to
+     gcs_bucket: The name of the GCS bucket to pull from
+     file_prefix: File name prefix used to identify which GCS blobs to fetch"""
+  client = storage.Client()
```

Comment:
Can you reuse the load_values_as_dataframe util function?

Reply:
I originally didn't use it because I wanted to avoid incurring a bunch of API calls getting the bucket, but I just split it into two functions so that the code can be shared.